hotsos/core/checks.py

```diff
@@ -766,3 +766,19 @@ def core(self):
         # go fetch
         self.all
         return self._core_snaps
+
+
+class SSLCertificatesChecksBase(object):
+
+    def __init__(self, certificate):
+        """
+        @param certificate: Expects an SSLCertificate object.
+        """
+        self.certificate = certificate
+
+    @property
+    def certificate_expires_soon(self):
+        "returns True if certificate expires in less than 60 days"
```

Comment:
Can you use a normal doctring format i.e.

""" Returns True if certificate expires in less than 60 days  """

Reply:
fixed on 59fc4895706a98108a6fc7c588544aceb54b1787